controllers/etcdpeer_controller.go

```diff
@@ -180,6 +180,9 @@ func defineReplicaSet(peer etcdv1alpha1.EtcdPeer, etcdRepository string, log log
 	etcdContainer := corev1.Container{
 		Name:  appName,
 		Image: fmt.Sprintf("%s:v%s", etcdRepository, peer.Spec.Version),
+		Args: []string{
```

Comment:
I think the correct syntax is:
```
Command: "foo"
Args: []string{"--bar", "baz"}
```
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#define-a-command-and-arguments-when-you-create-a-pod

Reply:
Good point. Fixed now